src/AuthLDAP.php

```diff
@@ -1458,23 +1458,16 @@ public static function displayLdapFilter($target, $users = true)
         }
         $config_ldap->getFromDB($_SESSION['ldap_server']);
 
+        $filter_name1 = "condition";
+        $filter_name2 = "condition";
```

Comment:
Not sure for this one. With your change, you always reach the `BOTH` case. I guess you should let the original code in the switch as is, and declare both filter variables to null per default.

Reply:
done